gradle/libs.versions.toml

```diff
@@ -1,24 +1,24 @@
 [versions]
-android-gradlePlugin = "8.2.2"
-android-compileSdk = "34"
+android-gradlePlugin = "8.8.2"
```

Comment:
Not sure if it matters for this project, but the latest stable version that IJ supports is 8.7.3